scripts/createProjectFolderStructure.js

```diff
@@ -0,0 +1,101 @@
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+const directoryName = require('path').resolve(__dirname, '..');
+
+var logFile = fs.createWriteStream('folder-structure.md', { flags: 'w' });
+var logStdout = process.stdout;
+
+console.log = function buildingStructure() {
+    logFile.write(util.format.apply(null, arguments) + '\n');
+    logStdout.write(util.format.apply(null, arguments) + '\n');
+};
+console.error = console.log;
+
+var folderNamesArray = [];
+var level = 0;
+var spacing = '';
+var descriptionSpacing = '';
+var description = '';
+
+console.log('# Folder Structue\n\n```');
+function crawl(spacing, dir, level) {
+    var folderName = path.basename(dir);
+    folderNamesArray.push(folderName);
+
+    // if(folderName==="webpack-cli"){
+    //     description=""
+    // }
+
+    var structure = spacing + '├───' + folderName;
+    var structureLength = structure.length;
+    var descriptionSpacingLength = 50 - structureLength;
+    descriptionSpacing = '';
+    for (var i = 0; i < descriptionSpacingLength; i++) {
+        descriptionSpacing += ' ';
+    }
+
+    console.log(structure + descriptionSpacing + description);
+
+    var files = fs.readdirSync(dir);
+    for (var x in files) {
+        //ignored folders
+        const blacklist = {
+            '.git': true,
+            '.nyc_output': true,
+            coverage: true,
+            node_modules: true,
+            bin: true,
+            lib: true,
+        };
+        if (!blacklist[files[x]]) {
```

Comment:
It seems a better approach would be to have `blacklist` as an arrray.

```js
      const blacklist = [ 
        '.git',
        '.nyc_output',
         'coverage',
         'node_modules',
         'bin',
         'lib',
      ];

      if (!blacklist.includes(files[x])) {
        //
      }
```

Reply:
Yeah, this is better than creating an object